Sources/Tests/CMakeLists.txt

```diff
@@ -1,3 +1,14 @@
+if(NOT EXISTS "${CMAKE_SOURCE_DIR}/Sources/Tests/gtest/.git")
+    # gtest is included as a git submodule, and there's a chance the repo
+    # wasn't cloned with submodules recursively. In that case, we want to try
+    # to automatically initialize it.
+    find_package(Git REQUIRED QUIET)
```

Comment:
While this is in fact required, I think that the error message this will produce (eg something like `can't find package Git`) might be a little misleading. The problem isn't really that git is missing - rather that the submodule hasn't been initialized. I wonder, then, if we should raise a custom error if git isn't found when the submodule is not initialized.